modules/javafx.base/src/main/java/javafx/beans/Subscription.java

```diff
@@ -59,7 +59,8 @@ static Subscription combine(Subscription... subscriptions) {
     /**
      * Cancels this subscription, or does nothing if already cancelled.<p>
      *
-     * Implementors must ensure the implementation is idempotent.
+     * Implementors must ensure the implementation is idempotent (a no-op
+     * if called more than once).
```

Comment:
I now think this should be in an `@implSpec` tag since it specifies the implementation conditions.